iothub/device/samples/PnpDeviceSamples/TemperatureController/ComponentTemperatureControllerCustomSerializer.cs

```diff
@@ -65,6 +58,12 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
                 await _deviceClient.UpdateClientPropertiesAsync(propertiesToBeUpdated, cancellationToken);
                 _logger.LogDebug($"Property: Update - {propertiesToBeUpdated.GetSerializedString()}.");
             }
+            else
+            {
+                var tValue = properties.Get<ThermostatInitialValue>("initialValue", Thermostat2);
```

Comment:
TryGet?

Reply:
If you had that, they wouldn't need to call `Contains()` first, which is simpler from an end-user POV

Reply:
yes, this is getting updated to TryGet